lte/gateway/python/scripts/generate_oai_config.py

```diff
@@ -24,6 +24,7 @@
     get_ip_from_if,
     get_ip_from_if_cidr,
     get_ipv6_from_if,
+    get_if_ip_with_netmask,
```

Comment:
Run https://docs.magmacore.org/docs/next/lte/dev_unit_testing#format-agw for formatting